plugins/modules/vmware_guest_disk.py

```diff
@@ -79,14 +79,19 @@
      - '     - C(thin) thin disk'
      - '     - C(eagerzeroedthick) eagerzeroedthick disk'
      - '     - C(thick) thick disk'
+     - '     - C(rdm) raw disk mapping'
      - '     Default: C(thick) thick disk, no eagerzero.'
      - ' - C(disk_mode) (string): Type of disk mode. Valid values are:'
      - '     - C(persistent) Changes are immediately and permanently written to the virtual disk. This is default.'
      - '     - C(independent_persistent) Same as persistent, but not affected by snapshots.'
      - '     - C(independent_nonpersistent) Changes to virtual disk are made to a redo log and discarded at power off, but not affected by snapshots.'
+     - ' - C(rdm_path) (string): RDM Path. Required if disk type is raw. /vmfs/devices/naa.[alpahnumeric string]'
+     - ' - C(compatibility_mode) (string): Compatibility mode for raw devices. Valid values are:'
```

Comment:
Add a changelog entry [`here`](https://github.com/ansible-collections/vmware/tree/main/changelogs/fragments).

Reply:
changelog has been updated with new entry 449-vmware_guest_disk_rdm.yml. I didn't see in any of the ones shown how features are label so I labeled it as "added feature:"